pkg/cmd/server/origin/handlers.go

```diff
@@ -21,27 +21,35 @@ import (
 	"github.com/openshift/origin/pkg/authorization/authorizer"
 )
 
+// RootPaths is the list of root paths we serve under "/". We expose it here so that we can
+// access it from integration tests
+var RootPaths = unversioned.RootPaths{}
```

Comment:
@deads2k is there a better way of plumbing this through to the integration tests?